client/lib/feed-stream-store/index.js

```diff
@@ -54,6 +54,27 @@ function buildNamedKeyMaker( property ) {
 	};
 }
 
+const conversationsKeyMaker = ( function() {
+	const baseMaker = buildNamedKeyMaker( 'last_comment_date_gmt' );
+	return function keyMaker( post ) {
+		const key = baseMaker( post );
+		key.comments = post.comments;
```

Comment:
just stashes the set of comments that we'll want to show on the post key. i'm going to swap this out in the future, but best way to proceed for now.